src/cli/Command.cpp

```diff
@@ -61,12 +63,14 @@ void populateCommands()
     if (commands.isEmpty()) {
         commands.insert(QString("add"), new Add());
         commands.insert(QString("clip"), new Clip());
+        commands.insert(QString("diceware"), new Diceware());
         commands.insert(QString("edit"), new Edit());
         commands.insert(QString("estimate"), new Estimate());
         commands.insert(QString("extract"), new Extract());
         commands.insert(QString("locate"), new Locate());
         commands.insert(QString("ls"), new List());
         commands.insert(QString("merge"), new Merge());
+        commands.insert(QString("passgen"), new PassGen());
```

Comment:
@TheZ3ro what about `gen` or `generate`? We already have `estimate` so `generate` isn't more verbose.

Reply:
@louib IDK, `generate` and `estimate` both refer to password but when I read the other commands all refers to entry or database (`show`, `add`, `edit`...). 
Using `generate` feels like generating an entry instead of generating a password

What do you think?

Reply:
Personally I'd go with `estimate`, but your call.

Reply:
Renamed